lib/utils/concatAddress/index.ts

```diff
@@ -2,6 +2,7 @@
  * @example
  * // 0x123…456
  */
-export const concatAddress = (address: string) => {
-  return address.slice(0, 5) + "…" + address.slice(address.length - 3);
+export const concatAddress = (address: string, numbers: number = 5) => {
```

Comment:
If you run `npm run build-carbonmark` locally you will see that this is what's causing typescript to be angry. This fixes it:

```suggestion
export const concatAddress = (address: string, numbers = 5) => {
```